src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class WinningLottoTest {
+    private Lotto lotto;
+    private List<Integer> lastWinningNumbers;
+
+    @BeforeEach
+    void setUp() {
+        lotto = new Lotto(List.of(
+                new LottoNumbers(List.of(1, 3, 5, 7, 19, 41)), // 4개 일치
```

Comment:
```suggestion
                new LottoNumbers(1, 3, 5, 7, 19, 41), // 4개 일치
```
LottoNumbers에 가변인자를 받는 생성자를 추가해 위와 같이 구현하면 어떨까?